plugins/webp-uploads/settings.php

```diff
@@ -48,45 +77,41 @@ function webp_uploads_add_media_settings_field() {
 add_action( 'admin_init', 'webp_uploads_add_media_settings_field' );
 
 /**
- * Renders the settings field for the 'perflab_generate_webp_and_jpeg' setting.
+ * Renders the settings field for the 'perflab_generate_avif_and_webp' setting.
  *
- * @since 1.0.0
+ * @since n.e.x.t
  */
-function webp_uploads_generate_webp_jpeg_setting_callback() {
-	if ( ! is_multisite() ) {
-		?>
-			</td>
-			<td class="td-full">
-		<?php
-	}
+function webp_uploads_generate_avif_webp_setting_callback() {
+
+	$selected = get_option( 'perflab_generate_avif_and_webp', 'avif' );
```

Comment:
AFAIK no default value is needed here since the default is defined where the setting is registered.

```suggestion
	$selected = get_option( 'perflab_generate_avif_and_webp' );
```